ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -1886,46 +1886,58 @@ class EclCmdZapGen : public EclCmdCommon
     }
     virtual int processCMD()
     {
-        Owned<IClientWsWorkunits> client = createCmdClient(WsWorkunits, *this);
-
-        Owned<IClientWUCreateZAPInfoRequest> zapReq = client->createWUCreateZAPInfoRequest();
-        zapReq->setWuid(optWuid.get());
-        zapReq->setIncludeThorSlaveLog(optIncThorSlave);
-        zapReq->setProblemDescription(optProblemDesc.get());
-
-        Owned<IClientWUCreateZAPInfoResponse> zapResp = client->WUCreateZAPInfo(zapReq);
-        int ret = outputMultiExceptionsEx(zapResp->getExceptions());
-        if (ret == 0)
+        if(optServer.isEmpty())
```

Comment:
Should these two checks happen in the finalizeOptions() method?  And simply return false rather than throw?

Reply:
Will move them to the finalizeOptions().